cves/kernel/CVE-2014-0206.yml

```diff
@@ -452,7 +446,9 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+  The failure here was a lapse. Failing to remember to vaildate input.
```

Comment:
This portion was meant to be "thoughtful" so you could maybe add to it or elaborate on it further and talk about the importance of such issues.